src/integrationTest/resources/db/testmigration/V1_18__alter_tables_add_columns.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE court_venue ADD COLUMN welsh_court_name varchar(256);
```

Comment:
Any chance we can make `varchar` into `VARCHAR` and the same for the other datatypes? 😄

Reply:
sure, will change that - thanks